pkg/tinydtls/Makefile

```diff
@@ -1,7 +1,9 @@
 PKG_NAME=tinydtls
 PKG_URL=https://github.com/rfuentess/TinyDTLS.git
-# PKG_VERSION=RIOT-OS
-PKG_VERSION=d06ad84d3e8bd3e86f7557faee34b68d5f8c0129
+PKG_VERSION=master
```

Comment:
My idea behind using the master branch instead of a specific commit was to be able to be working with the package TinyDTLS without requiring to do a PR to RIOT. As the TinyDTLS package can still be improved for RIOT. This is  why there was an extra commented line for a testing branch.

However, if this is a very bad course of action for RIOT. Then, I'll modify it to original approach.